.travis.yml

```diff
@@ -1,14 +1,19 @@
 language: erlang
 otp_release:
-   - 17.0-rc1
-   - R16B03-1
-   - R16B03
-   - R16B02
-   - R16B01
-   - R15B03
-   - R15B02
-   - R15B01
-   - R14B04
-   - R14B03
-   - R14B02 
-script: "make test"
\ No newline at end of file
+   - 19.0
+   - 19.1
+   - 19.2
+   - 19.3
+   - 20.0
+   - 20.1
+   - 20.2
+   - 20.3
+   - 21.0
+   - 21.1
+   - 21.2
+   - 21.3
+   - 22.0
+   - 22.1
+   - 22.2
+   - 22.3
```

Comment:
^ We should add a `23.0` target here, given the objective of this PR.

Reply:
I searched for it, couldn't find it. Where's the canonical list?

Reply:
There isn't one, I think.

For new versions, I usually [poll google](https://www.google.com/search?q=erlang+23+travis) until [something](https://travis-ci.community/t/erlang-23-0-released/8606) pops out confirming there is an image available.

Reply:
I'd found https://docs.travis-ci.com/user/languages/erlang/#choosing-otp-releases-to-test-against and it seems to point to `kerl`, so I'm simply adding 23.0 as per your recommendation (then we'll check the results, after Travis CI is available).